### ISSUE TITLE
🐛 [Bug] Error compiling torch.nn.BatchNorm1d on Docker built on top of branch release/ngc/22.01

### ISSUE THREAD

**Issue by chophilip21** [NVIDIA/Torch-TensorRT#889]
##  Bug Description

Previously I have encountered Docker build issues for your Torch-TensorRT Docker image, so I have checked out the branch release/ngc/22.01 according to instructions in #875, and this worked out fine. When I run Docker, I do get some warnings regarding CUDA version mismatch (CUDA 11.5 vs 11.6), but the container itself runs without any other issues.

However, when I am trying to compile a Pytorch CNN based model by following instructions in the [tutorial](https://nvidia.github.io/Torch-TensorRT/tutorials/getting_started_with_python_api.html), I have been getting exact same error mentioned on issue #692 for BatchNorm1d which is supposed to have already been fixed. 

## To Reproduce

1. Pull Pytorch Docker 22.01, and follow the instructions on issue #875 to build a Docker image.
2. Try replicating #692. My model uses a few blocks like below

```
self.classifier_concat = nn.Sequential(
    nn.BatchNorm1d(1024 * 3),
    nn.Linear(1024 * 3, feature_size),
    nn.BatchNorm1d(feature_size),
    nn.ELU(inplace=True),
    nn.Linear(feature_size, classes_num),
)
```

My issue is identical to #692, and this issue should have already been merged to master, but perhaps on the branch release/ngc/22.01, this is not reflected. Is there any work around for this? I like using Docker because building from source is often headache. I hope there is some other solutions.

**Comment by narendasan** [comment]
We don't backport bug fixes to NGC branches since there's a new one each month. I'd recommend that you take master and cherry-pick the few patches needed to base on 22.01.

**Comment by chophilip21** [comment]
I see. I have cherry picked one of the branches that have fixes for the batch norm, and things are working there.
